### PR TITLE
Update Coop.ch.xml

### PR DESCRIPTION
**Pull request by J0WI**
updated domain

### CODE REVIEW

File: src/chrome/content/rules/Coop.ch.xml [hunk 1]
```diff
@@ -14,8 +14,6 @@ Fetch error: http://impo.ch/ => https://www.impo.ch/: Redirect for 'http://www.i
 Fetch error: http://www.impo.ch/ => https://www.impo.ch/: Redirect for 'http://www.impo.ch/' missing Location
 Fetch error: http://christ-swiss.ch/ => https://www.christ-swiss.ch/: Redirect for 'http://www.christ-swiss.ch/' missing Location
 Fetch error: http://www.christ-swiss.ch/ => https://www.christ-swiss.ch/: Redirect for 'http://www.christ-swiss.ch/' missing Location
-Fetch error: http://the-body-shop.ch/ => https://www.the-body-shop.ch/: (51, "SSL: no alternative certificate subject name matches target host name 'www.the-body-shop.ch'")
-Fetch error: http://www.the-body-shop.ch/ => https://www.the-body-shop.ch/: (51, "SSL: no alternative certificate subject name matches target host name 'www.the-body-shop.ch'")
 -->
 <ruleset name="Coop" default_off='failed ruleset test'>
```
Comment (jsha) [inline src/chrome/content/rules/Coop.ch.xml]: Change this to 'Needs ruleset tests' or remove it and add the necessary tests.

**Comment by J0WI** [comment]
Finally! All tests passed.

**Comment by jsha** [comment]
There are a test url and a rule for www.coop.ch, but it is not included in the list of target hosts. Please remove them.

This should have been caught by the checker, I filed an issue: https://github.com/hiviah/https-everywhere-checker/issues/

**Comment by J0WI** [comment]
fixed

**Pull request merged**